README.adoc

```diff
@@ -2,188 +2,184 @@
+:clienta-link: https://start.spring.io/#!type=gradle-project&language=java&packaging=jar&groupId=com.example&artifactId=servicea&name=servicea&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.servicea&dependencies=cloud-eureka,web
+:clientb-link: https://start.spring.io/#!type=gradle-project&language=java&packaging=jar&groupId=com.example&artifactId=serviceb&name=serviceb&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.serviceb&dependencies=cloud-eureka,web
 
-NOTE: For convenience, we have provided build files (a `pom.xml` file and a `build.gradle`
-file) at the top of the project (one directory above the `service` and `client`
-directories) that you can use to build both projects at once. We also added the Maven and
-Gradle wrappers there.
+You can use the following links for pre-initialized projects from Spring Initializr:
 
-:server-link: https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.5.5&packaging=jar&jvmVersion=11&groupId=com.example&artifactId=service-registration-and-discovery&name=service-registration-and-discovery&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.service-registration-and-discovery&dependencies=cloud-eureka-server
-:client-link: https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.5.5&packaging=jar&jvmVersion=11&groupId=com.example&artifactId=service-registration-and-discovery-client&name=service-registration-and-discovery-client&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.service-registration-and-discovery-client&dependencies=cloud-eureka-server,cloud-eureka
+- {server-link}[Eureka Server^]
+- {clienta-link}[Service A^]
+- {clientb-link}[Service B^]
 
-You can use this {server-link}[pre-initialized project] (for the service application) or this {client-link}[pre-initialized project] (for the client application) and click Generate to download a ZIP file. This project is configured to fit the examples in this tutorial.
+NOTE: Due to the number of services in this guide, only the solutions are provided in the GitHub repository. If you'd like to start from scratch, please use generate the blank projects using the links above or using https://start.spring.io[Spring Initializr] as defined below.
 
 To manually initialize the project:
 
 . Navigate to https://start.spring.io.
 This service pulls in all the dependencies you need for an application and does most of the setup for you.
 . Choose either Gradle or Maven and the language you want to use. This guide assumes that you chose Java.
-. Click *Dependencies* and select *Eureka Server* for the service application and *Eureka Server* and *Eureka Discovery Client* for the client application.
+. Click *Dependencies* and select *Eureka Server* for the server application. Select *Eureka Discovery Client* and *Spring Web* for the client applications.
```

Comment:
There should be two sets of instructions here, one for the server and one for the clients.

Reply:
I made the change. No objections having 2 lists would increase the clarity. I originally opted for a single list, as 90% of the instructions are the same and I don't think many users will follow the start.spring.io path anyhow.
I can see an argument either way, so I made the change. If it looks too bloated we can always revert.